### PR TITLE
Rewrite config as a native golang structure

### PR DESCRIPTION
**Pull request by dshulyak**
I believe that such representation is much more obvious and doesn't require "ugly" go-bindata.

### CODE REVIEW

**Comment by dshulyak** [comment]
I also plan to remove web3js from our repository. And all other objects (keys mainly) doesn't need to be bundled with go-bindata at all

**Review by themue — APPROVED** [review]
Nice idea, definitely better that way.

**Comment by themue** [comment]
Other changes as parts of this PR or own PRs? Would prefer the latter.

**Comment by dshulyak** [comment]
> Other changes as parts of this PR or own PRs? Would prefer the latter.

me too, i will probably create a couple of small bounties

File: geth/params/cluster.go [hunk 1]
```diff
@@ -0,0 +1,55 @@
+package params
+
+type subClusterData struct {
+	Number      int      `json:"number"`
```
Comment (adambabik) [inline geth/params/cluster.go:4]: It looks like we do not use `Number` and `Hash`.

File: geth/params/config.go [hunk 2]
```diff
@@ -555,42 +555,19 @@ func (c *NodeConfig) updateClusterConfig() error {
...
-	type clusterData struct {
-		NetworkID int            `json:"networkID"`
-		Prod      subClusterData `json:"prod"`
-		Dev       subClusterData `json:"dev"`
-	}
-
-	var configFile []byte
-	var err error
-
+	var clusters []clusterData
```
Comment (adambabik) [inline geth/params/config.go:558]: We can initialize it with `defaultClusters`.

File: geth/params/cluster.go [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+	Hash        string   `json:"hash"`
+	StaticNodes []string `json:"staticnodes"`
+}
+
+type clusterData struct {
```
Comment (adambabik) [inline geth/params/cluster.go:9]: We may also think about flattening this structure:
```go
type cluster struct {
  NetworkID int `json:"networkID"`
  Prod bool `json:"production"` // or custom type based on int; we may have dev, prod, test
  StaticNodes []string `json:"staticnodes"`
}
```

**Review by adambabik — APPROVED** [review]

File: geth/params/config.go (hunk 2 above)
Comment (dshulyak) [reply geth/params/config.go:558]: i think we can overwrite defaultClusters array this way

File: geth/params/cluster.go (hunk 3 above)
Comment (dshulyak) [reply geth/params/cluster.go:9]: going to add bootnodes to this structure in the upcoming prs, and i will include simplification in the same pr

**Pull request merged**